compiler.rst

```diff
@@ -419,7 +504,28 @@ Important Files
 
     asdl_c.py
         "Generate C code from an ASDL description."  Generates
-        :file:`Python/Python-ast.c` and :file:`Include/Python-ast.h`.
+        :file:`Python/Python-ast.c` and :file:`Include/Internal/pycore_ast.h`.
```

Comment:
I like these descriptions but the header files get refactored quite a lot (specially the ast ones) so I am a bit scared that this gets out of date, but is still an improvement so I say: let's merge it.